js/books.js

```diff
@@ -0,0 +1,62 @@
+import Storage from './localStorage';
+import Book from './book';
+
+const form = document.querySelector('.form');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const booksContainer = document.querySelector('[data-list]');
+
+let booksArr = Storage.getStorage();
+
+class UI {
+  static displayData() {
+    const displayData = booksArr.map((item) => {
+      const { id, title, author } = item;
+      return `
+        <div class="book">
+          <p>${title}</p>
+          <p>${author}</p>
+          <button class="remove" data-id=${id}>remove</button>
+        </div>
+      `;
+    });
+    booksContainer.innerHTML = displayData.join(' ');
+  }
+
+  static removeBook() {
+    booksContainer.addEventListener('click', (e) => {
+      if (e.target.classList.contains('remove')) {
+        // remove book from ui
+        e.target.parentElement.remove();
+      }
+      // remove from array
+      const btnId = e.target.dataset.id;
+      UI.removeArrayBook(btnId);
+    });
+  }
+
+  static removeArrayBook(id) {
+    booksArr = booksArr.filter((item) => item.id !== +id);
+    Storage.addToStorage(booksArr);
+  }
+}
+
+window.addEventListener('DOMContentLoaded', () => {
+  UI.displayData();
+  UI.removeBook();
+});
+
```

Comment:
- [ ] Great job adding the `methods` to add and remove books. However the `add` and `remove` are not working fine. Please make sure the add and remove bookwork as expected.

[Screencast from 01-12-2023 06:47:43 PM.webm](https://user-images.githubusercontent.com/79058364/212130721-e8cee5ba-9683-45ef-b8f1-18bf6a5ba52d.webm)